pkg/controllers/disruption/suite_test.go

```diff
@@ -512,6 +595,18 @@ func ExpectTriggerVerifyAction(wg *sync.WaitGroup) {
 	}()
 }
 
+// ExpectTaintedNodes will assert the number of nodes and tainted nodes in the cluster and return the tainted nodes.
+func ExpectTaintedNodes(ctx context.Context, c client.Client, numNodes, numTaintedNodes int) ([]*v1.Node, []*v1.Node) {
```

Comment:
Is it worth just having these as separate functions? One function that asserts the number of tainted nodes, another that asserts the number of nodes. Come to think of it, I think that we have a function that asserts the number of nodes already

Reply:
I could go either way. I added it in here because the previous revision I believe you asked if it could make more sense as an ExpectTaintedNodes() function. I can pull out the nodes though